snap/snapcraft.yaml

```diff
@@ -1,17 +1,96 @@
-name: my-snap-name # you probably want to 'snapcraft register <name>'
-base: core24 # the base snap is the execution environment for this snap
-version: '0.1' # just for humans, typically '1.2+git' or '1.3.2'
-summary: Single-line elevator pitch for your amazing snap # 79 char long summary
+name: intel-npu-driver
+base: core22
```

Comment:
Not a technical challenge, I've just been trying to keep my build environment consistent with what I've tested directly on a host. Basically I've just been trying to limit changes to the build and runtime environments until now.

However, I agree that we should target `core24` now. I'd prefer to put that in a follow-up PR so we capture a working state for `core22` since that's already been tested. Sound okay to you?

Reply:
ok